### DISCUSSION TITLE
Whether the Address field can be configured with multiple instances

### DISCUSSION THREAD

**Discussion by uglyliu** [vectordotdev/vector#20029] [Q&A]
My config as follewers

---
data_dir: /data/svr/vector/data
timezone: "local"
api:
  enabled: false
  address: "[::]:8686"
  playground: false
sources:
  file1:
    include:
      - "/data/log/**/*.log"
    type: file
    fingerprint:
      strategy: "device_and_inode"
    ignore_checkpoints: true
    read_from: end
  internal_metrics:
    type: internal_metrics
sinks:
  prom_exporter:
    type: prometheus_exporter
    inputs:
      - internal_metrics
    address: "[::]:9090"
  vector1:
    type: vector
    inputs:
      - file1
    address: "10.35.2.213:8688"
    version: "2"

**Comment by jszwedko** [comment]
I'm not sure what question you are asking here 🤔 Could you restate it?

**Comment by uglyliu** [comment]
> I'm not sure what question you are asking here 🤔 Could you restate it?

For example, I have deployed multiple vector instances and I want to configure multiple IPs in adress. Can I do this?

address: "10.35.2.213:8688"

**Reply by jszwedko** [reply]
Ah I see, no, unfortunately you can only configure one address. The typical recommendation is to put a load balancer in-between. You could also use DNS to load balance. https://github.com/vectordotdev/vector/issues/18249 is tracking enhancements in this area.

**Reply by uglyliu** [reply]
> Ah I see, no, unfortunately you can only configure one address. The typical recommendation is to put a load balancer in-between. You could also use DNS to load balance. https://github.com/vectordotdev/vector/issues/18249 is tracking enhancements in this area.

Hmm, yes, I currently have an nlb load balancing installed in front。In this case, the logs are randomly placed on the disk of the vector server. if I mount remote storage, the logs of multiple services will be stored in one file. Will there be a risk of losing logs when writing? I don’t know how the vector placement works.
